src/Test/Perf/runner.csx

```diff
@@ -0,0 +1,15 @@
+#load ".\util.csx"
+
+using System.Collections.Generic;
+using System.IO;
+
+var myDir = MyWorkingDirectory();
+var skip = new HashSet<string> {
+    Path.Combine(myDir, "runner.csx"),
+    Path.Combine(myDir, "util.csx"),
```

Comment:
I would suggest that we put the util.csx in a wellknown subfolder (like "util") and exclude it that way.  When I navigate to the top level folder, I'd like to just see the .csx files that are meaningful for me to invoke as a developer (setup.csx, runner.csx, etc).